app/i18n/en/user.json

```diff
@@ -33,5 +33,7 @@
     },
     "noActivity": "No action recorded yet.",
     "browsingAnalyzer": "Browsing analyzer",
-    "reputation": "Reputation"
+    "reputation": "Reputation",
+    "iAgreeAndAcceptedPrivacy": "By clicking \"Sign up\", I agree that I have read and accepted ",
```

Comment:
Nit: Maybe using parameter would be easier to handle in case the button text is updated?

Reply:
This isn't the way to do it, and it is in many jurisdictions legally void.
It also apes bad practice, by being indistinguishable for the same reason people don't read.
So people don't read, and now they don't read and think this is bad software.
Do each point as a bulletpoint, or where you have to pick an answer.
All have to be checked/correct.